crates/api_models/src/enums.rs

```diff
@@ -558,12 +558,16 @@ pub enum RoutableConnectors {
     Worldpay,
 }
 
-#[derive(Debug, serde::Deserialize, serde::Serialize, Clone)]
+#[derive(Debug, serde::Deserialize, serde::Serialize, Clone, ToSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum SupportedWallets {
+    /// Paypal wallet is supported to get the session object
     Paypal,
+    /// Applepay wallet is supported to get the session object
     ApplePay,
+    /// Klarna wallet is supported to get the session object
     Klarna,
+    /// Gpay wallet is supported to get the session object
     Gpay,
```

Comment:
Most of these wallet names are self-explanatory. Add a doc comment along the lines of "Wallets which support obtaining session object" to the enum instead.

```rust
/// Wallets which support obtaining session object
pub enum SupportedWallets {
    Paypal,
    ApplePay,
    Klarna,
    Gpay,
}
```